packages/console/src/pages/SignInExperience/components/index.module.scss

```diff
@@ -3,6 +3,7 @@
 .title {
   color: var(--color-neutral-variant-60);
   font: var(--font-subhead-cap);
+  letter-spacing: 0.1em;
```

Comment:
We don't use `em` in other place, is this on purpose?

Reply:
According to the design file in Figma, this value should be 10% of the font size.

Reply:
font size is fixed (--font-subhead-cap), so we can still use a fixed "px" here.

Reply:
`em` is good though, just be caution to introduce new unit. @logto-io/eng

Reply:
per discussion, it's ok to use `em`, and we add a subhead css mixin to make it safer.
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1654152409797829